src/sv-pipeline/04_variant_resolution/scripts/merge_vcfs.py

```diff
@@ -35,6 +47,12 @@ def dedup_records(records):
     curr_record = records[0]
     for record in records[1:]:
         if records_match(curr_record, record):
+            # keep more informative ALT field
```

Comment:
Look at this again, I think we should add a little more description to what's going on here, ie the intention that we assume the more informative ALT contains more colons such as `INS:MEI` or `DUP:TANDEM`